pkg/sources/docker/docker_test.go

```diff
@@ -0,0 +1,51 @@
+package docker
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/context"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/credentialspb"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/sourcespb"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/sources"
+)
+
+func TestDockerImageScan(t *testing.T) {
```

Comment:
**Nit (optional):** This should probably be marked as an integration test since it's doing an actual scan of a remote image.

Reply:
Done!